### PR TITLE
inconsistent object repair in test_osd_replicated_inconsistency_scenario.py

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Modify logic for inconsistent object repair in test_osd_replicated_inconsistency_scenario.py
Jir Task - 1. https://issues.redhat.com/browse/RHCEPHQE-16167
               2. https://issues.redhat.com/browse/RHCEPHQE-16168

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
Pass Log- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F9YT65

**Comment by SrinivasaBharath** [comment]
Log- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-9P2EX9

**Comment by SrinivasaBharath** [comment]
Inconsistent OSD epoch test case logs-
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-59IJAD
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UESHO3
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JCKISU
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HWFHC0
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NPI90B
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SUH48W
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SUH48W

**Review by pdhiran — APPROVED** [review]
Looks good with minor comments, which will be taken care in the next PR.

File: tests/rados/test_osd_replicated_inconsistency_scenario.py [hunk 1]
```diff
@@ -68,8 +71,16 @@ def run(ceph_cluster, **kw):
                 "run the tests"
             )
             return 1
+        mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
```
Comment (pdhiran) [inline tests/rados/test_osd_replicated_inconsistency_scenario.py:74]: Make setting of the debug levels optional, based on params sent from suite file.

**Pull request merged**